lib/rules/lowercase_title.js

```diff
@@ -0,0 +1,67 @@
+const { isEmpty: _isEmpty, has: _has, keys: _keys } = require("lodash");
+const Rule = require("./Rule");
+
+module.exports = class LowercaseTitle extends Rule {
+    static meta = {
+        ruleId: "lowercase_title",
+        message: "Use lowercase title",
+        type: "off",
+        location: {},
+        hasFix: true,
+    };
+
+    static run(ast, config) {
+        if (_isEmpty(ast?.feature)) return [];
+
+        return new LowercaseTitle(ast, config).execute();
+    }
+
+    execute() {
+        this.checkTitle(this._ast);
+
+        return this.getProblems();
+    }
+
+    checkTitle(astObject) {
+        const keyword = _keys(astObject).shift();
+        if (["rule", "feature", "background", "scenario"].includes(keyword)) {
+            if (astObject[keyword].name.length !== 0) {
+                if (
+                    astObject[keyword].name[0] ===
+                    astObject[keyword].name[0].toUpperCase()
```

Comment:
This only checks the first character.

What about if someone writes:

`Feature: upload Files and Folders`

There are no test cases for that sort of thing.

Reply:
```suggestion
                    astObject[keyword].name !==
                    astObject[keyword].name.toLowerCase()
```

Will a change like this work?

Reply:
This rule is a tricky one. There are two options:
1. check only  the first character (current)
2. enforce that the sentence must be in lowercase (ignore if a whole word is in uppercase - acronym)

Reply:
we can do `enforce that the whole sentence/title must be in lowercase` for now then refine it later on

Reply:
I thought the rule must only check if the starting letter is in lowercase. 
Okay I will check for the whole sentence.